.github/workflows/release_docker.yml

```
@@ -4,10 +4,13 @@ on:
   workflow_call:
     inputs:
       ARCHITECTURE:
-        description: "The architecture to build for. Either 'x86_64' or 'arm64'"
+        description: "The architecture to build for. Either 'amd64' (for x86_64) or 'arm64'"
         required: true
-        default: "x86_64"
-        type: string
+        default: "amd64"
```

actionlint (1.7.11)

[error] input "ARCHITECTURE" of workflow_call event has the default value "amd64", but it is also required. if an input is marked as required, its default value will never be used

(events)